pkg/cli/session/session.go

```diff
@@ -178,69 +176,14 @@ func (e *BackplaneSession) Start() error {
 	cmd.Dir = e.Path
 	err = cmd.Run()
 	if err != nil {
-		return fmt.Errorf("error while running cmd. error %v", err)
-	}
-
-	err = e.killChildren()
-	if err != nil {
-		return err
+		fmt.Println("error while running cmd.")
 	}
 
 	fmt.Printf("Exited Backplane session \n")
 
 	return nil
 }
 
-// killChildren delete all pds in .killpds file
-func (e *BackplaneSession) killChildren() error {
```

Comment:
Is it safe to remove this func?

Reply:
Yeah. We don't track all processes running on the spawn child SHELL yet